stdlib/3.7/dataclasses.pyi

```diff
@@ -31,20 +31,22 @@ class Field(Generic[_T]):
     metadata: Optional[Mapping[str, Any]]
 
 
+# NOTE: Actual return type is 'Field[_T]', but we want to help type checkers
```

Comment:
Would it be better to just return Any? I'm worried we'll break some use case where people manually work with `Field` objects.

Reply:
Yes, but `_T` is used in args (and can probably be used for inference), so `Any` is also not optimal. I would rather break a less common use case, then later we could try to find a more permanent solution. Btw, `attrs` have the same signature for `attr.ib`, and no one complained so far (`attr` is supported for few months already).